### PR TITLE
First version

### PR DESCRIPTION
**Pull request by akm**
This is the first PR of this product.

## Main structs and interfaces

- watcherHandler (struct which has methods as [echo](https://echo.labstack.com/) handlers )
- Processor (interface)
  - Watcher (struct which implements Processor interface)
- Watch (struct for datastore)
- Notifier (interface)
    - GCSProxyNotifier (struct)

### Dependencies

```
[watcherHandler] ---> [Processor] <|---- [Watcher] 
[Watcher] -----> [Watch]
[Watcher] -----> [Notifier] <|---- [GCSProxyNotifier]
```

## Reviewers

- @nagachika 
- @taigou 
- @ko1nksm 
- @guemon 
- @totty 
- @minimum2scp

### CODE REVIEW

File: VERSION [hunk 1]
```diff
@@ -0,0 +1 @@
+20170111-02-witout-cron
```
Comment (minimum2scp) [inline VERSION]: Perhaps, s/witout/without/ ?

File: cron.yaml [hunk 2]
```diff
@@ -0,0 +1,4 @@
+cron:
+# - description: "Watch every 1 minute"
+#   url: /watches/refresh
```
Comment (nagachika) [inline cron.yaml:3]: Cron settings are commented out. Is it intended?

Comment (akm) [reply cron.yaml:3]: @nagachika Yes, it is. I'm going to activate these lines after some tests.
Actually I'd like to set cron settings on deploy. Is it possible?

File: glide.yaml [hunk 3]
```diff
@@ -0,0 +1,24 @@
+package: .
```
Comment (minimum2scp) [inline glide.yaml]: `glide install` says

```
[WARN]	The name listed in the config file (.) does not match the current location (github.com/groovenauts/blocks-gcs-watcher)
```

so, `package: .` should be `package: github.com/groovenauts/blocks-gcs-watcher`

**Review by totty — APPROVED** [review]

**Review by minimum2scp — COMMENTED** [review]
Missing `glide.lock` file.
Is it intended? (if so, it should be added into .gitignore)

**Review by taigou — APPROVED** [review]

File: VERSION (hunk 1 above)
Comment (akm) [reply VERSION]: @minimum2scp Thanks! I fixed it.

File: glide.yaml (hunk 3 above)
Comment (akm) [reply glide.yaml]: @minimum2scp Fixed it. Thank you.

**Comment by akm** [comment]
@minimum2scp I've just forgotten adding `glide.lock`, so I added it. Thanks a lot!

**Review by minimum2scp — APPROVED** [review]

**Review by nagachika — APPROVED** [review]

**Review by ko1nksm — APPROVED** [review]

**Review by guemon — APPROVED** [review]

**Pull request merged**